android/app/src/main/AndroidManifest.xml

```diff
@@ -99,6 +99,11 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <data android:mimeType="video/*" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.SEND" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="*/*" />
```

Comment:
why we need it when we have `text/*`, `document/*`, `video/*`, `image/*`

Reply:
Because */* allow any kind of mimeType even txt, odt, ....

Reply:
how about remove the unnecessary intent :))